catalog/video/video-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -33,9 +33,9 @@
                 type="String" default="" />
 
         <AD
-                description="Maximum file size (bytes) before rollover. Must be >=1."
+                description="Maximum file size (megabytes) before rollover. Must be >=1."
```

Comment:
If we don't make the max file size a double then we should add a note here that whole numbers only are allowed.